package.json

```diff
@@ -104,17 +104,18 @@
     "check-licenses": "wp-scripts check-licenses --production",
     "deploy": "grunt deploy",
     "dev": "wp-scripts start",
+    "diff-develop": "git diff develop --diff-filter=d --name-only",
```

Comment:
Perhaps `ls-changed-files` would be more appropriate.

Reply:
Alternatively, it may be better to just avoid having a separate command for this, and to instead just replace the instances of `npm run diff-develop` below with just the `git diff` command since it is short, and invoking `git` directly will reduce the need to fire up a subshell. If making this as fast as possible then this would be a consideration.

Reply:
Sure, applied in https://github.com/ampproject/amp-wp/pull/4441/commits/83e21556ba66a5a33464a9486e883c6ca990c785